src/test/logical_query_plan/lqp_utils_test.cpp

```diff
@@ -144,4 +146,28 @@ TEST_F(LQPUtilsTest, LQPFindSubplanRoots) {
   EXPECT_EQ(roots[2], subselect_a_lqp);
 }
 
+TEST_F(LQPUtilsTest, GetModifiedTables) {
+  const auto read_only_lqp =
```

Comment:
Surround the construction of LQPs with `clang-format off/on` otherwise clang-format will add unnecessary line breaks that make thinks harder to read. Within the clang-format-off block, make sure everything is indented with two spaces.

Reply:
Do you have an example where format skrewed up the formatting. Haven't any cases where it really bothered me since the results looked good to me.

Reply:
I like this format for LQPs:
```
auto lqp =
PredicateNode::make(predicate,
  SomeOtherNode::make(param
    stored_table_node
```

and that's what's used everywhere else. clang-format like to abominate that into

```
auto lqp =
  PredicateNode::make(
    predicate,
    SomeOtherNode::make(
    param
      stored_table_node
```

and I cannot read that without feeling ill.

Reply:
But looking at line 150: isn't that what you want except from a tiny indention difference?

Reply:
almost, I'd put "node_a" into a separate and stick to clean 2-spaces indentation. And I wouldn't indent the root node, but that's my preference